File: kotlin/PlanesAndroid/app/src/test/java/PlaneRoundTest.java
import com.google.common.truth.Truth;
import com.planes.common.PlaneRound;

import org.junit.Test;
import static com.google.common.truth.Truth.assertThat;

public class PlaneRoundTest {

    @Test
    public void PlaneRound_playerGuessAlreadyMade() {
        PlaneRound pr = new PlaneRound(10, 10, 3);
        assertThat(pr.playerGuessAlreadyMade(0, 0)).isEqualTo(0);
    }
}

File: kotlin/PlanesAndroid/app/src/test/java/PlaneRoundTest.kt
package com.planes.test
import com.google.common.truth.Truth
import com.planes.single_player_engine.PlaneRound
import org.junit.Test

class PlaneRoundTest {
    @Test
    fun PlaneRound_playerGuessAlreadyMade() {
        val pr = PlaneRound(10, 10, 3)
        Truth.assertThat(pr.playerGuessAlreadyMade(0, 0)).isEqualTo(0)
    }
}